src/LiveComponent/src/EventListener/LiveComponentSubscriber.php

```diff
@@ -148,6 +152,31 @@ public function onKernelController(ControllerEvent $event): void
 
         $request->attributes->set('_mounted_component', $mounted);
 
+        // autowire live file arguments
+        if ($request->files->count()) {
+            $allFiles = $request->files->all();
+            $accessor = PropertyAccess::createPropertyAccessor();
+            foreach (LiveFileArg::liveFileArgs($component, $action) as $parameter => $fileArg) {
+                $path = $fileArg->getPropertyPath();
```

Comment:
This method uses the `$this->name` property on `LiveFileArg`. But that is (correctly) optional. If it's not set, fallback to `$parameter`, just like we do with `LiveArg`. I believe you were planning this already :)

Reply:
Logic for that is inside `liveFileArgs` method already :+1: